frontend/packages/dev-console/locales/en/devconsole.json

```diff
@@ -262,6 +262,17 @@
   "Create Source-to-Image Application": "Create Source-to-Image Application",
   "Git repo URL": "Git repo URL",
   "Create Sample Application": "Create Sample Application",
+  "JAR": "JAR",
+  "JAR file": "JAR file",
+  "Drag a file here or browse to upload": "Drag a file here or browse to upload",
+  "Optional Java commands": "Optional Java commands",
+  "Java commands are Application specific and can be added to customize your Application.": "Java commands are Application specific and can be added to customize your Application.",
```

Comment:
@lwrigh Please let us know your inputs

Reply:
<img width="851" alt="Screen Shot 2021-03-15 at 10 26 06 AM" src="https://user-images.githubusercontent.com/20227374/111168883-df11fd80-8578-11eb-8bb5-dd73e8cd5083.png">

Application is considered a proper noun according to the [OpenShift glossary](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/term_glossary.adoc) and we capitalize Application in other parts of the UI.

Reply:
@lwrigh the term `application` is overloaded here. The screenshot you refer to is the `application label` which is used to group workloads and related resources. Whereas the "java application" is specifically a single executable piece of code which will become a single running workload.

Furthermore, the examples in the glossary link you provided are all lower case.

Reply:
The [conventions](http://openshift.github.io/openshift-origin-design/conventions/documentation/capitalization.html) call for title case for proper nouns and application is considered a proper noun.

Reply:
I agree with the the conventions of capitalization. 
However I can find over 150 instances of `application` in console and the [OpenShift glossary for application](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/term_glossary.adoc#application) examples use lowercase. I'm not sure I've ever considered `application` to be a proper noun when describing a program. `Application` being title case when describing the grouping via label may be appropriate, but doesn't mean all instances of the word should be treated the same.

Reply:
If we are talking about the resource, it would be Application.  But in this case we aren't.  We refer to "deployments" in places where it is not capitalized.

IMO this should be lower case.

Reply:
Have updated to lower case